unlock-app/package.json

```diff
@@ -120,4 +120,4 @@
     "prettier-eslint-cli": "^4.7.1",
     "react-testing-library": "^5.2.3"
   }
-}
\ No newline at end of file
+}
```

Comment:
ack!

Reply:
Just have this change to package.json, which I will assume you'll squash

Reply:
Oups! Indeed!